crates/oro-client/src/client.rs

```diff
@@ -80,6 +154,32 @@ impl OroClientBuilder {
             client_uncached,
         }
     }
+
+    #[cfg(not(target_arch = "wasm32"))]
+    fn get_no_proxy(&self) -> Option<NoProxy> {
+        if let Some(ref no_proxy_conf) = self.proxy_config.no_proxy_domain {
+            if no_proxy_conf != "NO_PROXY" || no_proxy_conf != "" {
+                Some(NoProxy::from_string(no_proxy_conf));
```

Comment:
I'm surprised this compiles? [`NoProxy::from_string` already returns `Option<NoProxy>`](https://docs.rs/reqwest/latest/reqwest/struct.NoProxy.html#method.from_string)

Reply:
Because I added semicolon accidentally, so ... 🤣